### DISCUSSION TITLE
How to check results

### DISCUSSION THREAD

**Discussion by donvman** [aryainjas/Microllect#5] [General]
![Screenshot_2022-03-28-17-36-56-477_com.realvnc.viewer.android.jpg](https://user-images.githubusercontent.com/100040715/160445839-c3e6205a-2ce7-4862-a312-22031c9957ee.jpg)

Where can I check results with balance???

**Comment by aryainjas** [comment]
If an address with non-zero balance found,first you will see that in the bottom of gui.
At the address with balance section.
There will be a .txt file at the path you extracted the repo.
Good luck and be aware of things you do.
